src/css/application.css

```diff
@@ -266,6 +266,23 @@ body {
   background-color: var(--color-gray);
 }
 
+.librarypicker__item.dashboard__menu-item_active::before {
```

Comment:
So adding properties to a conjunction of two classes like this, I think, diverges from the spirit of [BEM](http://getbem.com/introduction/), which strenuously avoids any interdependencies between different block definitions.

I would propose instead adding a `dashboard__menu-item_checked` and corresponding `dashboard__menu-item_unchecked`, with the latter just adding the padding. We’d only use those modifiers in the `<LibraryPickerItem>` component.

That would also fix another issue, which is that currently the padding is applied to _all_ dashboard menu items, including ones that never take a checkbox:

![image](https://cloud.githubusercontent.com/assets/14214/24838560/28b95596-1d18-11e7-8578-e5561418c544.png)